sky/templates/kubernetes-ray.yml.j2

```diff
@@ -279,6 +279,7 @@ available_node_types:
             # Identifies the SSH jump pod used by this pod. Used in life cycle management of the ssh jump pod.
             skypilot-ssh-jump: {{k8s_ssh_jump_name}}
             skypilot-user: {{ user }}
+            skypilot-original-cluster-name: {{cluster_name}}
```

Comment:
We probably don't need to do this if we look up our state db to map `cluster_name_on_cloud` -> `cluster_name`?

Reply:
Updated the solution to get from DB.